crates/apollo-smith/src/description.rs

```diff
@@ -51,6 +51,13 @@ pub enum StringValue {
     Line(String),
 }
 
+#[cfg(feature = "parser-impl")]
+impl From<apollo_parser::ast::StringValue> for StringValue {
+    fn from(val: apollo_parser::ast::StringValue) -> Self {
+        Self::from(Into::<String>::into(val))
```

Comment:
This `Into::<String>::into()` business is a bit awkward, i wonder if we could `impl From<StringValue> for String` instead? or provide a separate named method that returns the unquoted contents of a StringValue?

Reply:
We already have `impl Into<String> for ast::StringValue` [here](https://github.com/apollographql/apollo-rs/blob/e8a4540f88b5d504a36380d82a8dfd40787a4a50/crates/apollo-parser/src/ast/node_ext.rs).`From`s are usually nicer, but that requires us to create a syntax node with an unknown source, which perhaps is a bit too tricky? What do you think?